graphrag/vector_stores/typing.py

```diff
@@ -8,15 +8,15 @@
 
 from .azure_ai_search import AzureAISearch
 from .lancedb import LanceDBVectorStore
-from .kustodb import KustoDBVectorStore
+from .kusto import KustoVectorStore
```

Comment:
I originally had it as kusto without the db. I can add that back if the team wants. I have no preference.